cl/search/models.py

```diff
@@ -2624,6 +2624,13 @@ class OpinionCluster(AbstractDateTimeModel):
         blank=True,
         db_index=True,
     )
+    filepath_pdf_harvard = models.FileField(
+        help_text="The case PDF from the Caselaw Access Project for this cluster",
+        upload_to=make_upload_path,
+        storage=IncrementingAWSMediaStorage(),
+        blank=True,
+        null=True,
```

Comment:
You don't usually want null=True on string fields in Django (it's a convention). So:

```suggestion
```